sphinx_gallery/gen_rst.py

```diff
@@ -498,11 +498,8 @@ def generate_file_rst(fname, target_dir, src_dir, gallery_conf):
     ref_fname = example_file.replace(os.path.sep, '_')
     example_rst = """\n\n.. _sphx_glr_{0}:\n\n""".format(ref_fname)
 
-    if not fname.startswith('plot'):
-        convert_func = dict(code=codestr2rst, text=text2string)
-        for blabel, bcontent in script_blocks:
-            example_rst += convert_func[blabel](bcontent) + '\n'
-    else:
+    filename_pattern = gallery_conf.get('filename_pattern', 'plot_*')
+    if fnmatch.fnmatch(fname, filename_pattern):
```

Comment:
Just in case people forget about this, because it is buried deep in a folded outdated diff, I am putting here my earlier comment back in favor regex instead of glob:

> I'd rather have re.search on the absolute path of the example with regex pattern as I mentioned in the associated issue. This way it is more flexible. I would argue that this is a reasonably advanced feature so it is not too much to expect the person using it to know about regex.